lib/runners/processor.rb

```diff
@@ -218,15 +218,18 @@ def read_report_file(file_path = report_file)
     class InvalidXML < SystemError; end
 
     def read_xml(text)
-      root = REXML::Document.new(text).root
+      root = Nokogiri::XML(text) do |config|
```

Comment:
[suggestion] `root` is confusing. `doc` is better?

```suggestion
      doc = Nokogiri::XML(text) do |config|
```

Reply:
Fixed it on https://github.com/sider/runners/pull/2427/commits/328a69815f402560307656153c4a0a9fc9b89a49